Resources/Prototypes/Entities/Objects/Fun/Instruments/base_instruments.yml

```diff
@@ -71,6 +71,8 @@
         - BulletImpassable
   - type: StaticPrice
     price: 300
+  - type: VendPrice # Frontier
```

Comment:
Please stop adding this to things, for the 100000th time. IT BREAKS OUR ECONOMY

Reply:
But I did fix the test to drop itself so it cannot under sell ever.
If I bump the static it will up alot of the ships sell prices that has instruments on them.

The all point of me making vendprice take lower priority over market price was to make it impossible to break the economy.